packages/element-library/src/gif/display.js

```diff
@@ -27,24 +26,7 @@ import { StoryPropTypes } from '@googleforcreators/elements';
  * Internal dependencies
  */
 import MediaDisplay from '../media/display';
-import { getBackgroundStyle, videoWithScale } from '../video/util';
-
-const Video = styled.video`
-  position: absolute;
-  max-width: initial;
-  max-height: initial;
-  ${videoWithScale}
-`;
-
-const Image = styled.img`
-  position: absolute;
-  max-height: initial;
-  object-fit: contain;
-  width: ${({ width }) => `${width}px`};
-  left: ${({ offsetX }) => `${-offsetX}px`};
-  top: ${({ offsetY }) => `${-offsetY}px`};
-  max-width: ${({ isBackground }) => (isBackground ? 'initial' : null)};
-`;
+import { getBackgroundStyle, Video, VideoImage } from '../video/util';
```

Comment:
Since these are shared between the gif and the video element, this is a good opportunity to move them to `media/util.js` instead.